tests/gocase/unit/command/command_test.go

```diff
@@ -35,13 +35,6 @@ func TestCommand(t *testing.T) {
 	rdb := srv.NewClient()
 	defer func() { require.NoError(t, rdb.Close()) }()
 
-	t.Run("Kvrocks supports 183 commands currently", func(t *testing.T) {
-		r := rdb.Do(ctx, "COMMAND", "COUNT")
-		v, err := r.Int()
-		require.NoError(t, err)
-		require.Equal(t, 183, v)
-	})
-
```

Comment:
@git-hulk do we remove this test intentionally?

Reply:
Yes, as said in comment: https://github.com/apache/incubator-kvrocks/pull/782#issuecomment-1252227937, this test case is not much help for us and it causes conflicts frequently, so I think we should remove.

Reply:
Get it. Although, the code conflict doesn't relate to this test but (1) I'm porting scripting tests (2) this PR added new scripting tests XD.